core/provers.go

```diff
@@ -48,9 +48,10 @@ type LightClient interface {
 
 // FinalityAware provides the capability to determine the finality of the chain
 type FinalityAware interface {
-	// GetLatestFinalizedHeader returns the latest finalized header on this chain
-	// The returned header is expected to be the latest one of headers that can be verified by the light client
-	GetLatestFinalizedHeader() (latestFinalizedHeader Header, err error)
+	// GetFinalizedHeader returns the finalized header on this chain corresponding to `height`.
+	// If `height` is nil, this function returns the latest finalized header.
```

Comment:
@siburu Could you fix this comment as s/nil/0/ ?

Reply:
Thank you! I fixed it.